SocketIOClientSwift/SocketEngine.swift

```diff
@@ -625,59 +592,59 @@ public final class SocketEngine: NSObject, WebSocketDelegate, SocketLogClient {
             client?.engineDidClose("Ping timeout")
             return
         }
-        
+
         ++pongsMissed
-        write("", withType: PacketType.PING, withData: nil)
+        write("", withType: PacketType.Ping, withData: nil)
     }
-    
+
     /// Send polling message.
     /// Only call on emitQueue
-    private func sendPollMessage(var msg:String, withType type:PacketType,
-        datas:ContiguousArray<NSData>? = nil) {
-            SocketLogger.log("Sending poll: %@ as type: %@", client: self, args: msg, type.rawValue)
-            
+    private func sendPollMessage(var msg: String, withType type: PacketType,
+        datas:[NSData]? = nil) {
+            Logger.log("Sending poll: %@ as type: %@", type: logType, args: msg, type.rawValue)
+
             doubleEncodeUTF8(&msg)
             let strMsg = "\(type.rawValue)\(msg)"
-            
+
             postWait.append(strMsg)
-            
-            if datas != nil {
-                for data in datas! {
-                    let (nilData, b64Data) = createBinaryDataForSend(data)
-                    
+
+            if let datas = datas {
+                for data in datas {
```

Comment:
You can simplify this by doing `for data in datas ?? []`. That way you reduce the indentation level.